### PR TITLE
Added Firefox omnibar search. :mag:

### PR DESCRIPTION
**Pull request by darenju**
Added a simple form and some JavaScript to allow users to use Firefox
built-in search engine.
Right-clicking on the search field and selecting « Add a keyword search
engine » allows one to type something like:

`emoji thumb`

This will load the app and show thumb emojis.

### CODE REVIEW

**Comment by muan** [comment]
Dang, sorry about the delayed. I'm not a firefox user so haven't been able to get myself setup to dig into this. Please bear with me.

**Comment by darenju** [comment]
No problem about the delay.
It should work as expected with no side effect.

**Comment by muan** [comment]
Hi! Thanks for much for contributing. This is definitely something I'd like to add to the site. A couple of things though:
1. The changes you made should live in `speedy.coffee` where the search functions live.
2. Having to redirect from `?search=%s` is kind of a pain, and ugly :cry:. I see that Chrome supports something like this:
   
   image
   
   Is there nothing like this in Firefox? I tried googling but found nothing. Was hoping there might be a special attribute or metatag for such thing. Like how [`opensearch.xml`](https://github.com/muan/emoji/blob/c5e8e457a95495a2a265d5a6d5f6458c39317833/opensearch.xml#L5) effects the custom search engines.
   
   I guess if there isn't a way then this is :ok_hand:.

File: _assets/stuff.coffee [hunk 1]
```diff
@@ -11,6 +21,10 @@ $(document).on 'emoji:ready', ->
     clip.on "complete", (_, args) -> $("<div class=alert></div>").text("Copied " + args.text).appendTo("body").fadeIn().delay(1000).fadeOut()
     $(".emoji-code").attr("readonly", "readonly")
 
+  $('#search-form').submit (e) ->
```
Comment (muan) [inline _assets/stuff.coffee:24]: Let's move this out of the `emoji:ready` check.

**Comment by darenju** [comment]
I will take a long at the second point as soon as I get home, where I'll have my MBA. But it seems right clicking on the field doesn't show the context menu option if there's no form.
I will invest on a more elegant way to "redirect", or go directly to `#search`.
Thanks for reviewing my PR, by the way.

**Pull request closed without merging**